frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -195,8 +195,7 @@ class BitBox02 extends Component<Props, State> {
                 errorText: undefined,
             });
             if (status === 'initialized' && unlockOnly && showWizard) {
-                // bitbox is unlocked, now route to / and wait for incoming accounts
-                route('/', true);
+                route('/account-summary', true);
```

Comment:
Will this work nicely with AOPP where, after unlocking, it proceeds to the addr. verification flow?

Reply:
Before it routed to `/` and then to `/account-summary`,
with this change it just goes directly to `/account-summary`.

AOPP just covers everything no matter what it is, expect if the user needs help unlocking/settingup, then aopp can minimize into a small bar at the top or something else.